### ISSUE TITLE
debounce is not a function when changing uiengine.config.js

### ISSUE THREAD

**Issue by naltatis** [dennisreimann/uiengine#16]
uiengine crashes with the following error, when I change the uiengine.config.js file in the root directory of the project.

```
$ uiengine build --serve --watch
[...]
Watching files...
./node_modules/@uiengine/core/lib/uiengine.js:66
    return debounce('handleFileChange', () => {
           ^

TypeError: debounce is not a function
    at handleFileChange (./node_modules/@uiengine/core/lib/uiengine.js:66:12)
    at FSWatcher.chokidar.watch.on.on.on.filePath (./node_modules/@uiengine/core/lib/uiengine.js:82:206)
    at emitTwo (events.js:126:13)
    at FSWatcher.emit (events.js:214:7)
    at FSWatcher.<anonymous> (./node_modules/chokidar/index.js:199:15)
    at awfEmit (./node_modules/chokidar/index.js:216:9)
    at FSWatcher.<anonymous> (./node_modules/chokidar/index.js:319:9)
    at FSReqWrap.oncomplete (fs.js:153:5)
```

**Comment by dennisreimann** [comment]
Alright, I‘ll fix this with the upcoming release :)

**Comment by dennisreimann** [comment]
Fixed with v0.19.1 – thanks for the hint! (now this part is also covered with tests :))
